REST Bindings/openapi.yaml

```diff
@@ -2818,7 +2817,7 @@ components:
                     type: string
                     format: uri
                     enum:
-                      - epcisException:ImplementationException
+                      - epcisException:IUnsupportedMediaTypeException
```

Comment:
```suggestion
                      - epcisException:UnsupportedMediaTypeException
```

Reply:
Hi @jmcanterafonseca-iota this would require a discussion as currently the `UnsupportedMediaTypeException` does not exist in the spec. I suggest we discuss today.

Reply:
Unsupported Media Type exception would presumably map to an HTTP 415 response code ( see for example https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/415  - although the documentation there only mentions inspection of the data payload or the headers Content-Type:  or Content-Encoding:  
We'd need to check very carefully whether this or 415 is appropriate in situations where the client requests (via Accept: header) a media type that the server can't support.  From what I read at https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/415 and at https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.13  it's not clear that the server can use it to reject a media type specified by the client via the Accept: header - instead I get the impression that it's for a server to reject an actual data format sent by the client - so that could be applicable more on the capture interface or if the query client sent its query in an unsupported format (e.g. attempted to send an XML-formatted query to the REST interface of an implementation that only supported JSON-formatted queries).